packages/vaadin-themable-mixin/test/css-modules.test.js

```diff
@@ -0,0 +1,221 @@
+import { expect } from '@vaadin/chai-plugins';
+import { fixtureSync, oneEvent } from '@vaadin/testing-helpers';
+import { parseStyleSheets } from '../src/css-modules.js';
+
+const BASE_PATH = import.meta.url.split('/').slice(0, -1).join('/');
+
+describe('CSS modules', () => {
+  it('should extract modules defined via @media rules', () => {
+    fixtureSync(`
+      <style>
+        @media lumo_base-field {
+          #label {
+            color: gray;
+          }
+        }
+      </style>
+      <style>
```

Comment:
Is there a need to use two separate style tags here? If not, I think they could be combined.

Reply:
Separate style tags ensure that modules and tags are extracted from _all stylesheets_ that are passed to `parseStyleSheets`.